template.yaml

```diff
@@ -303,11 +421,10 @@ Resources:
           ALLOWED_ORIGIN: '*'
           TABLE_NAME: !Ref DoiRequestsTableName
           INDEX_NAME: !Ref DoiRequestsIndexName
+          ASSUMED_ROLE_ARN: !GetAtt AssumedRole.Arn
           API_HOST: !Ref CustomDomain
-          API_SCHEME: !Ref HttpScheme
-      Policies:
-        - DynamoDBCrudPolicy:
-            TableName: !Ref DoiRequestsTableName
+          API_SCHEME: https
```

Comment:
Use parameter with default to https as a poor man constant?

Reply:
Allows to override if we want to run locally.

Reply:
fixed